Can_Node_Handler.cpp

```diff
@@ -85,6 +89,13 @@ void Can_Node_Handler::handleMessage(Frame& message) {
 
   // Write torque commands
   writeThrottleMessages(outputTorque);
+
+  int16_t raw_RPM = Store().readMotorRpm(0);
+  int actual_motor_RPM = raw_RPM/4000; 
+  int wheel_RPM = actual_motor_RPM*2.17;//using gear ratio to convert Motor RPM to Wheel RPM
+  if(wheel_RPM>2500 && analogThrottle<50){
+    writeBrakeMessages(50);//Test Brake value when throttle unpressed
```

Comment:
Seems like you could potentially send a positive torque and a negative torque at the same time (see line 91). That would be bad, right?